gulpfile.ts

```diff
@@ -54,34 +65,43 @@ gulp.task('styles', () => (
       use: [autoprefixer({ browsers: ['last 2 versions'] })],
       compress: true,
     }))
-    .pipe(gulp.dest('dist'))
+    .pipe(gulp.dest('dist/app'))
 ));
 
 gulp.task('__build', gulp.series(
   'clean',
-  gulp.parallel('cards', 'static', 'styles')
+  'typescript',
+  gulp.parallel('cards', 'static', 'styles'),
 ));
 
-
 // Utility functions -------------------------------------------------------------------------------
 
-function buildCardObject(filename, meta, contents) {
+interface CardMeta {
+  authors: string[];
+  categories: string[];
+  created: string;
+  drugs: string|null; // FIXME: This should be an array
```

Comment:
No need... Using the `pain-management` card as an example, here's the difference between rendering a string and rendering an array....

### String
```yml
title: Initial Pain Management Options
authors:
    - Dr. Nick Koch
    - Dr. Sergey Motov
created: 2015/10/23
updates: null
categories:
    - Pharmacology
drugs: acetaminophen, Ibuprofen, Naproxen, Ketorolac, Indomethacin, lidocaine, lidoderm, Bupivacaine, Triamcinolone, Gabapentin, Pregabalin, Ketamine, Oxycodone, Hydrocodone, Hydromorphone, Morphine, Fentanyl, naloxone
```

### Array
```yml
title: Initial Pain Management Options
authors:
    - Dr. Nick Koch
    - Dr. Sergey Motov
created: 2015/10/23
updates: null
categories:
    - Pharmacology
drugs: [acetaminophen, Ibuprofen, Naproxen, Ketorolac, Indomethacin, lidocaine, lidoderm, Bupivacaine, Triamcinolone, Gabapentin, Pregabalin, Ketamine, Oxycodone, Hydrocodone, Hydromorphone, Morphine, Fentanyl, naloxone]
```